vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh

```diff
@@ -198,7 +213,7 @@ maxupdateskew 100.0
 rtcsync
 
 # Settings come from: https://docs.microsoft.com/en-us/azure/virtual-machines/linux/time-sync
-refclock PHC /dev/ptp0 poll 3 dpoll -2 offset 0
+refclock PHC /dev/ptp_hyperv poll 3 dpoll -2 offset 0 stratum 2
```

Comment:
we dont need to change anything for azurelinux ?

Reply:
from PR description sounds like this wouldn't ever be needed on AzureLinux since we only build V2/V3, and we've stopped building CBLMarinerV1 sometime ago